airflow/providers/google/cloud/operators/dataprep.py

```diff
@@ -49,6 +48,68 @@ def __init__(self, *, job_id: int, **kwargs) -> None:
 
     def execute(self, context: Dict):
         self.log.info("Fetching data for job with id: %d ...", self.job_id)
-        hook = GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
+        hook = GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id",)
```

Comment:
The connection should be configurable via operator interface

Reply:
Yes, indeed. Thanks.